modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1089,6 +1092,521 @@ RETURN wrote
 ----
 
 
+[role=enterprise-edition]
+[[constraints-examples-node-property-type]]
+== Node property type constraints
+
+A node property type constraint ensures that for all nodes with a certain label and property, the property is of the required property type.
+
+The allowed property types for the constraints are:
+
+* `BOOLEAN`
+* `STRING`
+* `INTEGER`
+* `FLOAT`
+* `DATE`
+* `LOCAL TIME`
+* `ZONED TIME`
+* `LOCAL DATETIME`
+* `ZONED DATETIME`
+* `DURATION`
+* `POINT`
+
+For all available Cypher types see xref::values-and-types/property-structural-constructed.adoc#types-synonyms[Types and their synonyms].
+
+* xref::constraints/examples.adoc#constraints-create-a-node-property-type-constraint[]
+* xref::constraints/examples.adoc#constraints-create-a-node-property-type-constraint-if-not-exist[]
+* xref::constraints/examples.adoc#constraints-create-an-already-existing-node-property-type-constraint[]
+* xref::constraints/examples.adoc#constraints-create-a-different-than-existing-node-property-type-constraint[]
+* xref::constraints/examples.adoc#constraints-create-a-node-that-complies-with-a-property-type-constraint[]
+* xref::constraints/examples.adoc#constraints-create-a-node-that-violates-a-property-type-constraint[]
+* xref::constraints/examples.adoc#constraints-fail-to-create-a-property-type-constraint-due-to-existing-node[]
+
+
+[discrete]
+[[constraints-create-a-node-property-type-constraint]]
+=== Create a node property type constraint
+
+When creating a node property type constraint, a name can be provided.
```

Comment:
I realise you are following what is already on the page, but for indexes we say that best practice is to give a name upon creation. Should we not do the same for constraints? (If yes, I can update for the other instances of this sentence on the page).  

```suggestion
When creating a node property type constraint, it is recommended to provide a constraint name.
```

Reply:
as with everything else, this is currently consistent with the other constraints sections and should therefore be updated for all of them

Reply:
@Hunterness - agreed that the page should be consistent, but that may not be within the scope of this PR. Given that 5.9 is already out, I believe the best action now is to accept (where you think appropriate) the stylistic changes, and then I can come back to this page and make sure the other examples follow with these new style suggestions.

Reply:
✅